jetbrains-core/src/software/aws/toolkits/jetbrains/core/credentials/profiles/ProfileCredentialProviderFactory.kt

```diff
@@ -289,6 +289,9 @@ class ProfileCredentialProviderFactory(private val ssoCache: SsoCache = diskCach
         val mfaSerial = profile.property(ProfileProperty.MFA_SERIAL)
             .orElse(null)
 
+        // https://docs.aws.amazon.com/sdkref/latest/guide/setting-global-duration_seconds.html
+        val durationSecs = profile.property(ProfileProperty.DURATION_SECONDS).map { it.toIntOrNull() }.orElse(null) ?: 3600
```

Comment:
Could we assign the value 3600 to a variable(`defaultDuration` maybe)?

Reply:
Normally I would agree, but I think in this case I want to keep everyone together since it ties back to that commented docs page